src/Server/TCPHandler.cpp

```diff
@@ -1133,6 +1133,12 @@ void TCPHandler::receiveQuery()
     }
     query_context->applySettingsChanges(settings_changes);
 
+    /// Disable function name normalization it's a secondary query.
```

Comment:
Some explanation needed, e.g. "because they were already normalized on initiator node".

Reply:
It's needed for old initiator new shard compatibility. Comments will be refined.

Reply:
Haha... yes, a comment is needed :)